decidim-module-census_connector/app/forms/decidim/census_connector/verifications/census/data_form.rb

```diff
@@ -26,7 +26,6 @@ class DataForm < Decidim::Form
           attribute :postal_code, String
 
           validates :document_scope_id, presence: true, unless: :local_document?
-          validates :document_scope, presence: true, unless: :verified?
```

Comment:
I used this to enforce that the given scope_id is a valid organization scope, don't you think it is important to do that to avoid unexpected errors or security issues?

Reply:
You're right. I blindly remove all validations and deferred to census, but census doesn't know about organizations so that should be kept indeed.

Reply:
@leio10 I think it's enough to scope the `find_by` queries, not only by the `*scope_id`, but to the current organization as well. Makes sense?

Reply:
Maybe I should clarify a bit more. I assume the scope selectors are already scoped to the organization scopes (valga la redundancia), so the validation in the form object doesn't really protect us from anything there. How could the invalid input be entered? Where and how would we show the error? We _do_ need protection if the (mis-)user skips the form and sends the request directly with invalid inputs. But for that case, scoping the queries should be enough I think.

Reply:
I was thinking on the user accidentally or intentionally sending wrong scope codes. But as you said, having all validations on Census allow us to simplify all this code, so this is not required. Sorry for making you to think about this, I don't know why I didn't see it clear before. :S

Reply:
Nono, your feedback is very useful! I changed the scope queries so that they are retrieve only scopes for the current organization :+1: